libs/silly-wisher-names/src/loadTrans.ts

```diff
@@ -0,0 +1,158 @@
+import fs = require('fs')
+// Load translation files from POEditor into the assets folder.
+const namesFilePath = `${__dirname}/../Translated/CharMemNames.csv`
+const localeDir = `${__dirname}/../assets/locales/`
+
+const raw = fs.readFileSync(namesFilePath).toString()
+
+const lines = raw.split('\r\n')
+
+lines.shift() // First line is langauge heading
+lines.pop() // Empty last line
+// console.log(lines)
+const languages = [
+  'en',
+  'chs',
+  'cht',
+  'ja',
+  'fr',
+  'it',
+  'de',
+  'es',
+  'ko',
+  'th',
+  'id',
+  'pt',
+  'ru',
+  'vi',
+  'tr',
+] as const
+type LangKey = (typeof languages)[number]
+const data = Object.fromEntries(languages.map((l) => [l, {}])) as Record<
+  LangKey,
+  { [key: string]: string }
+>
+
+const swKeyMap = {
+  aether: 'TravelerM',
+  albedo: 'Albedo',
+  alhaitham: 'Alhaitham',
+  aloy: 'Aloy',
+  amber: 'Amber',
+  itto: 'AratakiItto',
+  // 'baizhu':'Baizhu',
+  barbara: 'Barbara',
+  beidou: 'Beidou',
+  bennett: 'Bennett',
+  candace: 'Candace',
+  chongyun: 'Chongyun',
+  collei: 'Collei',
+  cyno: 'Cyno',
+  dehya: 'Dehya',
+  diluc: 'Diluc',
+  diona: 'Diona',
+  dori: 'Dori',
+  eula: 'Eula',
+  faruzan: 'Faruzan',
+  fischl: 'Fischl',
+  ganyu: 'Ganyu',
+  gorou: 'Gorou',
+  hutao: 'HuTao',
+  jean: 'Jean',
+  kazuha: 'KaedeharaKazuha',
+  kaeya: 'Kaeya',
+  ayaka: 'KamisatoAyaka',
+  ayato: 'KamisatoAyato',
+  keqing: 'Keqing',
+  klee: 'Klee',
+  sara: 'KujouSara',
+  kuki: 'KukiShinobu',
+  layla: 'Layla',
+  lisa: 'Lisa',
+  lumine: 'TravelerF',
+  mika: 'Mika',
+  mona: 'Mona',
+  nahida: 'Nahida',
+  nilou: 'Nilou',
+  ningguang: 'Ningguang',
+  noelle: 'Noelle',
+  qiqi: 'Qiqi',
+  raiden: 'RaidenShogun',
+  razor: 'Razor',
+  rosaria: 'Rosaria',
+  kokomi: 'SangonomiyaKokomi',
+  sayu: 'Sayu',
+  shenhe: 'Shenhe',
+  heizou: 'ShikanoinHeizou',
+  sucrose: 'Sucrose',
+  tartaglia: 'Tartaglia',
+  thoma: 'Thoma',
+  tighnari: 'Tighnari',
+  venti: 'Venti',
+  wanderer: 'Wanderer',
+  xiangling: 'Xiangling',
+  xiao: 'Xiao',
+  xingqiu: 'Xingqiu',
+  xinyan: 'Xinyan',
+  yae: 'YaeMiko',
+  yanfei: 'Yanfei',
+  yaoyao: 'Yaoyao',
+  yelan: 'Yelan',
+  yoimiya: 'Yoimiya',
+  yunjin: 'YunJin',
+  zhongli: 'Zhongli',
+} as const
+
+lines.forEach((line) => {
+  const [
+    skey,
+    en, // English (United States)
+    cht, // Chinese (Traditional)
+    chs, // Chinese(Simplified)
+    ja, // Japanese
+    fr, // French
+    it, // Italian
+    de, // German
+    es, // Spanish
+    ko, // Korean
+    th, // Thai
+    id, // indonesian
+    pt, // Portuguese(Brazil)
+    ru, // Russian
+    ,
+    // Polish
+    vi, // Vietnamese
+    tr, // Turkish // Arabic, Filipino / Tagalog
```

Comment:
These comments are confusing. Perhaps use underscore prefixed variables instead, e.g. `_po`

Reply:
yeah I had to fight with prettier, I think even `_po` has issue with linter

Reply:
Yea I think it is a typescript setting to allow the underscore throwaway variables. Similar thing was happening in datamine somewhere iirc. But the underscored variables are allowed in our frontend code